### PR TITLE
ROC-2782 Add show more, show less content box

### PR DESCRIPTION
**Pull request by robertparkinson**
https://tools.hmcts.net/jira/browse/ROC-2782

prototype example: 
https://hmcts-cmc-prototype.herokuapp.com/defendant/full/d-5/task-list/your-defence/defence-1

### CODE REVIEW

File: macros/responseStatus.njk [hunk 1]
```diff
@@ -42,11 +42,14 @@
   {% elseif claim.status === ClaimStatus.FREE_MEDIATION %}
     {{ t('{{ defendantName }} has rejected the claim. They’ve suggested free mediation on {{ respondedAt }}. You can accept or reject',
     { defendantName: claim.response.defendant.name, respondedAt: claim.respondedAt | date }) }}
+    <p><a href="/case/{{ claim.externalId }}/response/view-response">{{ t('View full response') }}</a></p>
```
Comment (timja) [inline macros/responseStatus.njk]: should use the paths variable for this and call `evaluateUri({ externalId: claim.externalId })` on it

File: macros/showMore.njk [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{% macro showMore(label, content) %}
+  <div class="show-more">
+    <div class="bold-small">{{ t(label) }}</div>
+    {% if longText.length > 400 %}
```
Comment (DamianDunajski) [inline macros/showMore.njk]: Where the `longText` variable is defined?

File: macros/showMore.njk [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+      <input id="more-less" type="checkbox">
+    {% endif %}
+    <p class="form-group-compound">{{ content }}</p>
+    {% if longText.length > 400 %}
+      <label class="link more" for="more-less">{{ t('Show more') }}</label>
+      <label class="link less" for="more-less">{{ t('Show less') }}</label>
```
Comment (DamianDunajski) [inline macros/showMore.njk]: Shouldn't one of these links be hidden by default?

**Comment by DamianDunajski** [comment]
@robertparkinson how will that work when JS is off, the prototype does not demonstrate it. Will the full content be displayed in such case?

File: macros/showMore.njk (hunk 3 above)
Comment (robertparkinson) [reply macros/showMore.njk]: this uses css to hide and show and does not rely on javascript. The css to control it is in the citizen frontend

File: macros/showMore.njk (hunk 2 above)
Comment (robertparkinson) [reply macros/showMore.njk]: forgot to change it when I pulled it out to the common-frontend. Updated now

**Comment by DamianDunajski** [comment]
It would be good to see the whole code including how this macro is being used to review it properly. 

@robertparkinson suggested we could put it initially in the frontend before we extract it to frontend-commons. Alternatively attaching some code snippet to the PR would work as well.

**Pull request closed without merging**